core/src/com/po/fuck/model/position/GeometryData.java

```diff
@@ -0,0 +1,66 @@
+package com.po.fuck.model.position;
+
+import com.badlogic.gdx.math.Vector2;
+
+public class GeometryData {
+    protected Vector2 position;
+    protected float rotation; /* in rads */
+    protected float width;
+    protected float height;
+
+    public GeometryData(){}
+
+    public GeometryData(Vector2 position, float width, float height, float rotation){
+        this.position = position.cpy();
+        this.rotation = rotation;
+        this.width = width;
+        this.height = height;
+    }
+
+    public GeometryData(GeometryData other){
+        this.position = other.position.cpy();
+        this.rotation = other.rotation;
+        this.width = other.width;
+        this.height = other.height;
+    }
+
+    public Vector2 getPosition(){
+        return position.cpy();
+    }
+
+    public void setPosition(Vector2 newPosition){
+        this.position = newPosition.cpy();
+    }
+
+    public float getWidth(){
+        return width;
+    }
+
+    public void setWidth(float width){
+        this.width = width;
+    }
```

Comment:
Why do we have public setters and getters, that do not perform any transfrom of the data, can't we just make a variable public ?

Reply:
Ok, I can set several checks for set.